react/components/FilterOptionTemplate.js

```diff
@@ -152,60 +152,53 @@ const FilterOptionTemplate = ({
 
   return (
     <div className={containerClassName}>
-      {!hasBeenViewed ? (
-        dummyElement
-      ) : (
-        <>
-          <div className={titleContainerClassName}>
-            <div
-              role="button"
-              tabIndex={collapsable ? 0 : undefined}
-              className={collapsable ? 'pointer' : ''}
-              onClick={() => collapsable && setOpen(!open)}
-              onKeyDown={handleKeyDown}
-              aria-disabled={!collapsable}
-            >
-              <div className={titleClassName}>
-                {title}
-                {collapsable && (
-                  <span
-                    className={classNames(
-                      handles.filterIcon,
-                      'flex items-center ph5 c-muted-3'
-                    )}
-                  >
-                    <IconCaret orientation={open ? 'up' : 'down'} size={14} />
-                  </span>
+      <div className={titleContainerClassName}>
```

Comment:
This entire section is just the removal of an indentation level and of this ternary